mmstudio/src/main/java/org/micromanager/display/DisplaySettings.java

```diff
@@ -197,6 +199,15 @@ interface Builder {
     */
    double getAutoscaleIgnoredPercentile();
 
+
+   /**
+    * When autoscaling, whether or not to take zero values into account.
+    * This can be useful for images that contain artificial zero values.
+    *
+    * @return Whether or not to ignore zeros in the
+    */
+   boolean ignoreZerosWhenAutoScaling();
```

Comment:
Since this is an API method, would be good to follow the convention of starting boolean-returning getters with `is`: maybe `isAutoscaleIgnoringZeors()`.
(No need to change the builder method name.)

Reply:
Good suggestions, done.